### PR TITLE
Open drawer all the way when keyboard is shown

### PR DESCRIPTION
**Pull request by vanwagonet**
refs: MBL-13660
affects: Student
release note: Fixed a bug where the keyboard hides the submission comment text field

### CODE REVIEW

**Comment by instructure-cx** [comment]
Scan submitted to Checkmarx

**Comment by instructure-cx** [comment]
### Checkmarx scan completed
[Full Scan Details](https://cxmgr.inssec.info/CxWebClient/ViewerMain.aspx?scanid=1014748&projectid=17)
#### Checkmarx Scan Summary
Severity|Count
---|---
High|0
Medium|0
Low|0
Informational|0

#### Violation Summary
Severity|Count
---|---

**Comment by inst-danger** [comment]
#### Release Note: 
Fixed a bug where the keyboard hides the submission comment text field

#### Affected Apps: Student

[MBL-13660](https://instructure.atlassian.net/browse/MBL-13660)

Coverage | New % | Master % | Delta
-------- | ----- | -------- | -----
**Canvas iOS** | 93.29% | 93.32% | -0.02%
<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

File: Student/Student/Submissions/SubmissionDetails/SubmissionDetailsViewController.swift [hunk 1]
```diff
@@ -44,6 +44,10 @@ class SubmissionDetailsViewController: UIViewController, SubmissionDetailsViewPr
         return controller
     }
 
+    deinit {
+        NotificationCenter.default.removeObserver(self)
```
Comment (armstrongnate) [inline Student/Student/Submissions/SubmissionDetails/SubmissionDetailsViewController.swift]: FYI you don't have to call this as of iOS 9+ with `addObserver(_:selector:name:object:)`

**Pull request merged**